src/components/search/search.hbs

```diff
@@ -1,8 +1,8 @@
 {{#is suffix "sm"}}
 <div class="bx--form-item">{{/is}}
-  <div data-search class="bx--search bx--search--{{suffix}}{{#if light}} bx--search--light{{/if}}" role="search">
+  <div data-search role="search" class="bx--search bx--search--{{suffix}}{{#if light}} bx--search--light{{/if}}">
     <label id="search-input-label-1" class="bx--label" for="search__input-1">Search</label>
-    <input class="bx--search-input" type="text" role="search" id="search__input-1" placeholder="Search" aria-labelledby="search-input-label-2">
+    <input class="bx--search-input" type="text" id="search__input-1" placeholder="Search" aria-labelledby="search-input-label-1">
```

Comment:
This is more-so an issue with the originally markup, but why do we have `label` with `for` and `aria-labelledby` set?